programs/framework/pe_dependencies/member.py

```
@@ -1068,3 +1068,43 @@ class SelfEmploymentIncomeBeforeLsrDependency(IncomeDependency):
+class InSecondarySchoolDependency(Member):
+    """
+    PolicyEngine's ``is_in_secondary_school`` — secondary school, or an equivalent level of
+    vocational or technical training.
+
+    The variable has no formula and no default, so PolicyEngine reads False for everyone
+    unless it is sent. Its consumers all use it as
+    ``where(is_in_secondary_school, student_age_limit, non_student_age_limit)``, so an
+    unsent input silently applies the lower limit.
+
+    Means high school or equivalent vocational training — distinct from PolicyEngine's
+    ``is_in_k12_school``, which covers all of K-12 and imputes ages 5–17.
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)